live/global.ini

```diff
@@ -6716,33 +6716,33 @@ Dlg_SC_bb_bb_factory_ORIG_scanners_being_scanned_0210=Schiff wurde gescannt.
+Dlg_SC_bb_bb_factory_ORIG_shields_back_25_0570=Hintere Schilde bei: 25%
+Dlg_SC_bb_bb_factory_ORIG_shields_back_50_0560=Hintere Schilde bei: 50%
+Dlg_SC_bb_bb_factory_ORIG_shields_back_75_0550=Hintere Schilde bei: 75%
+Dlg_SC_bb_bb_factory_ORIG_shields_bottom_0_0740=Untere Schilde erschöpft.
+Dlg_SC_bb_bb_factory_ORIG_shields_bottom_25_0730=Untere Schilde bei: 25%
+Dlg_SC_bb_bb_factory_ORIG_shields_bottom_50_0720=Untere Schilde bei: 50%
+Dlg_SC_bb_bb_factory_ORIG_shields_bottom_75_0710=Untere Schilde bei: 75%
 Dlg_SC_bb_bb_factory_ORIG_shields_charging_0507=Erneuerung der Schilde
 Dlg_SC_bb_bb_factory_ORIG_shields_collision_alert_0760=ANNÄHERUNGS-ALARM
-Dlg_SC_bb_bb_factory_ORIG_shields_front_0_0540=Vorderer Schild erschöpft.
-Dlg_SC_bb_bb_factory_ORIG_shields_front_25_0530=Vorderer Schild auf ein Viertel Stärke.
-Dlg_SC_bb_bb_factory_ORIG_shields_front_50_0520=Vorderer Schild auf halbe Stärke.
-Dlg_SC_bb_bb_factory_ORIG_shields_front_75_0510=Vorderer Schild bei 75% Stärke.
-Dlg_SC_bb_bb_factory_ORIG_shields_left_0_0620=Linker Schild erschöpft.
-Dlg_SC_bb_bb_factory_ORIG_shields_left_25_0610=Linker Schild auf ein Viertel Stärke.
-Dlg_SC_bb_bb_factory_ORIG_shields_left_50_0600=Linker Schild auf halbe Stärke.
-Dlg_SC_bb_bb_factory_ORIG_shields_left_75_0590=Linker Schild bei 75%.
-Dlg_SC_bb_bb_factory_ORIG_shields_right_0_0660=Rechter Schild erschöpft.
-Dlg_SC_bb_bb_factory_ORIG_shields_right_25_0650=Rechter Schild auf ein Viertel Stärke.
-Dlg_SC_bb_bb_factory_ORIG_shields_right_50_0640=Rechter Schild auf halbe Stärke.
-Dlg_SC_bb_bb_factory_ORIG_shields_right_75_0630=Rechter Schild bei 75%.
+Dlg_SC_bb_bb_factory_ORIG_shields_front_0_0540=Vordere Schilde erschöpft.
+Dlg_SC_bb_bb_factory_ORIG_shields_front_25_0530=Vordere Schilde bei: 25%
+Dlg_SC_bb_bb_factory_ORIG_shields_front_50_0520=Vordere Schilde bei: 50%
+Dlg_SC_bb_bb_factory_ORIG_shields_front_75_0510=Vordere Schilde bei: 75%
+Dlg_SC_bb_bb_factory_ORIG_shields_left_0_0620=Linke Schilde erschöpft.
+Dlg_SC_bb_bb_factory_ORIG_shields_left_25_0610=Linke Schilde bei: 25%
+Dlg_SC_bb_bb_factory_ORIG_shields_left_50_0600=Linke Schilde bei: 50%
+Dlg_SC_bb_bb_factory_ORIG_shields_left_75_0590=Linke Schilde bei: 75%
+Dlg_SC_bb_bb_factory_ORIG_shields_right_0_0660=Rechte Schilde erschöpft.
+Dlg_SC_bb_bb_factory_ORIG_shields_right_25_0650=Rechte Schilde bei: 25%
+Dlg_SC_bb_bb_factory_ORIG_shields_right_50_0640=Rechte Schilde bei: 50%
+Dlg_SC_bb_bb_factory_ORIG_shields_right_75_0630=Rechte Schilde bei: 75%
 Dlg_SC_bb_bb_factory_ORIG_shields_system_activated_0750=Einsatzbereit.
-Dlg_SC_bb_bb_factory_ORIG_shields_top_0_0700=Oberer Schild erschöpft.
-Dlg_SC_bb_bb_factory_ORIG_shields_top_25_0690=Oberer Schild auf ein Viertel Stärke.
-Dlg_SC_bb_bb_factory_ORIG_shields_top_50_0680=Oberer Schild auf halbe Stärke.
-Dlg_SC_bb_bb_factory_ORIG_shields_top_75_0670=Oberer Schild bei 75%.
+Dlg_SC_bb_bb_factory_ORIG_shields_top_0_0700=Oberere Schilde erschöpft.
```

Comment:
Obere

Reply:
Kannst das mal bitte selbst kurz ändern? Bin erst in der Nacht wieder zuhause